help-docs/FAQ/join-tg-group.md

```diff
@@ -36,38 +36,46 @@ For best results:
 
 1. Verify you received a legitimate invite link from `@collablandbot`.
 
-   - The invite will be a link in the format telegram.me/**collablandbot**?start=ABCD1234...
+   - Your admin will share the invite will be a link in the format telegram.me/**collablandbot**?start=ABCD1234...
    - If the link is not _**exactly**_ `collablandbot`, it is an imposter.
 
 2. Use the invite link generated by the bot.  
 
-3. Verify the `Community:` field is correct for the community you are trying to join.
+3. Click the `START` button
+
+4. Verify the `Community:` field is correct for the community you are trying to join.
 
 <div class="text--center">
   <img src={img3a} alt="Verify Community details" width="400" />
 </div>
 
-4. Click the "Connect Wallet" button from the bot.
+5. Click the "Connect Wallet" button from the bot.

```

Comment:
should we mention that you can "use existing wallets" or "add a new wallet" - and then "Connect Wallet"?
<img width="394" alt="image" src="https://github.com/abridged/collabland-dev/assets/56363630/5f72df30-baf5-4d17-b17c-ab564664101c">